include/tree_option.h

```diff
@@ -0,0 +1,40 @@
+
+
+#ifndef tree_option_h
+#define tree_option_h
+
+#include <string>
+// Include MenuOption
+#include <menu_option.h>
```

Comment:
menu_option.h is a local file which is in the same directory. Use #include "menu_option.h" (makes the preprocessor look for it in the same directory as this file).